bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/items/ItemUpdater.java

```diff
@@ -67,7 +79,14 @@ protected void receiveUpdate(ItemStateEvent updateEvent) {
                     }
                 }
                 if (isAccepted) {
+                    State oldState = item.getState();
                     item.setState(newState);
+                    if (oldState != null && !oldState.equals(newState)) {
```

Comment:
No, I think it should be changed to a check for `UnDefType.NULL` to cover the startup case I mentioned above.